libosrmc/osrmc.cc

```diff
@@ -29,6 +29,21 @@ struct osrmc_error final {
   std::string message;
 };
 
+void osrmc_error_from_json(osrm::json::Object* json, osrmc_error_t* error) try {
```

Comment:
We only use this function in the implementation, right? Then it should be ` static` .

Reply:
Great point! I'd forgotten about `static`. :-)